cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -211,6 +281,15 @@ public String apply(@Nullable String input) {
     return result;
   }
 
+  private Map<String, String> parseTagValuesAsMap(List<String> tags) {
```

Comment:
this should use parseTagValues or the other way - to keep the parsing code in one place..